### DISCUSSION TITLE
✅ Is payment per device or per user ?

### DISCUSSION THREAD

**Discussion by chmike** [isontheline/pro.webssh.net#177] [Q&A] [answered]
I have an iPhone for which I payed the upgrade to full version. 

A weird thing is that I have no feedback that I already payed. If I click on the shopping cart, it ask again for a payment or a refund. I guess I could pay again. Didn't try.

I also have an iPad, but it isn't upgraded. Do I have to pay the upgrade again for the iPad ?

**Comment by chmike** [comment, marked as answer]
I reply to myself since I found out. On the iPad, I have to click request a refund although I didn't pay anything on the iPad. Webssh on the iPad is then considered as an upgraded version.

I tried it because I have seen the same behavior with xterminal, another ssh client app, but not as good as webssh.

Don't know if this is a "feature", a loophole, or whatever. 

It is annoying that it is not clear if I'm supposed to pay per device or per user. In the later case, it is not clear what I should do to not pay per device.

**Reply by isontheline** [reply]
Bonjour Christophe 👋

Sorry for the delay...

> Do I have to pay the upgrade again for the iPad ?

No you only to paid once for a user for all devices linked to it

> Don't know if this is a "feature", a loophole, or whatever.

It's more like a "feature" but related to how Apple is handling purchases and restores on their side.

I want to thank you for your support of WebSSH !

Have a nice day ☀️
🇫🇷
